python/tk_multi_workfiles/file_list/user_filter_menu.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, parent):
 
         self._current_user_action = QtGui.QAction("Show My Files", self)
         self._current_user_action.setCheckable(True)
-        toggled_slot = lambda toggled, uid=self._current_user_id: self._on_user_toggled(uid, toggled)
+        toggled_slot = lambda toggled: self._on_user_toggled(self._current_user_id, toggled)
```

Comment:
Old version of PySide don't like methods with too many arguments, even if they are default arguments.